build/templates/_converters.py.mako

```diff
@@ -0,0 +1,32 @@
+<%
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+%>\
+from ${module_name} import visatype
+
+
+def _timedelta_converter(value, library_type, scaling):
+    if str(type(value)).find("'datetime.timedelta'") != -1:
+        scaled_value = value.total_seconds() * scaling
+    else:
+        scaled_value = value
+
+    if not library_type == visatype.ViReal64:  # ctype integer types don't convert to int from float so we need to
+        scaled_value = int(scaled_value)
```

Comment:
With timeouts, it's better to round up rather than floor the value. Of course it just happens that this conversion function is used with timeouts, but there's nothing about the generator's design that implies this is only for timeouts.

Do you see any problem with rounding down the scaled_value blindly?

Reply:
Also... what about ViReal32?
Maybe we should do the explicit cast no matter what (meaning, remove the if statement).

Reply:
None of our APIs use ViReal32.

We can't do this all the time. If the library type is a double, we need to leave it as is.